websites/M/Modrinth/presence.ts

```diff
@@ -0,0 +1,269 @@
+const presence = new Presence({
+		clientId: "1047217921685991462",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "https://i.imgur.com/GnMSCG4.png",
+			startTimestamp: browsingTimestamp,
+		},
+		{ pathname, hostname, href } = document.location,
+		pathSplit = pathname.split("/").filter(x => x);
+
```

Comment:
More compact code, for changelog, gallery, etc.
![image](https://user-images.githubusercontent.com/42322979/206899765-1c2cbd73-3a3b-41b7-8338-e195944ea5ff.png)
```suggestion
if (active?.textContent) presenceData.details = `Browsing ${pathSplit[0]} ${active?.textContent}`
else {
```